soc/riscv/riscv-ite/it8xxx2/Kconfig.soc

```diff
@@ -20,6 +20,18 @@ endchoice
 
 if SOC_IT8XXX2
 
+choice
```

Comment:
Please add a label to this choice.  `choice IT8XXX2_SERIES`

Reply:
Done.